code/processes/tailreader.q

```diff
@@ -7,6 +7,9 @@ currentpartition:@[value;`currentpartition;getpartition[]]
 basedir:raze (getenv`KDBTAIL),"/tailer",(string .ds.segmentid),"/"         /-define associated tailer base directory
 taildir:`$ basedir,string currentpartition;                                /-define tailDB direction
 
+/- log message if tailreader is started without datastriping activated
+if[not .ds.datastripe;.lg.e[`load;"Datastriping is disabled: terminating ",(string .proc.procname)," process."]];
+
```

Comment:
Does this not need to be at the top of the file so nothing is defined prior in case of an uncaught error such as .ds.sementid not being specified?